components/esp_dns/esp_dns_doh.c

```diff
@@ -0,0 +1,245 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "freertos/FreeRTOS.h"
```

Comment:
also , do we need freertos in this module?

Reply:
No. Removed.